src/Sushi.Mediakiwi.API/Services/ContentService.cs

```diff
@@ -302,6 +303,162 @@ private async Task<ICollection<ListGrid>> GetGridsAsync()
 
         #endregion Get Grids
 
+        #region Get Datalist
+
+        private async Task<ListGrid> GetDataListAsync(IComponentListTemplate listInstance)
+        {
+            ListGrid grid = new ListGrid();
+
+            List<ListDataColumnType> hiddenTypes = new List<ListDataColumnType>()
+            {
+                ListDataColumnType.Highlight,
+                ListDataColumnType.UniqueHighlightedIdentifier,
+                ListDataColumnType.UniqueIdentifier,
+                ListDataColumnType.APIOnly
+            };
+
+            #region Columns
+
+            foreach (var col in listInstance.wim.ListDataColumns.List.Where(x => x.Type != ListDataColumnType.APIExcluded))
+            {
+                grid.Columns.Add(new ListColumn()
+                {
+                    Align = (int)col.Alignment,
+                    helpText = col.Tooltip,
+                    IsAverage = col.Total == ListDataTotalType.Average,
+                    IsSum = col.Total == ListDataTotalType.Sum,
+                    IsHidden = hiddenTypes.Contains(col.Type),
+                    Prefix = col.ColumnValuePrefix,
+                    Suffix = col.ColumnValueSuffix,
+                    Title = col.ColumnName,
+                    Width = col.ColumnWidth
+                });
+            }
+
+            #endregion Columns
+
+            #region Layer Configuration
+
+
+            if (listInstance?.wim?.CurrentList?.Option_LayerResult == true && listInstance?.wim?.Page?.Body?.Grid?.LayerConfiguration == null)
+            {
+                listInstance.wim.Page.Body.Grid.SetClickLayer(new Grid.LayerSpecification(LayerSize.Normal));
+            }
+
+            if (listInstance?.wim?.Page?.Body?.Grid?.LayerConfiguration != null)
+            {
+                var config = listInstance.wim.Page.Body.Grid.LayerConfiguration;
+                grid.LayerConfiguration = new LayerConfiguration()
+                {
+                    HasScrollbar = config.HasScrolling.GetValueOrDefault(false),
+                    Height = config.Height.GetValueOrDefault(),
+                    HeightUnitType = config.IsHeightPercentage ? UnitTypeEnum.Percentage : UnitTypeEnum.Pixels,
+                    Title = config.Title,
+                    Width = config.Width.GetValueOrDefault(),
+                    WidthUnitType = config.IsWidthPercentage ? UnitTypeEnum.Percentage : UnitTypeEnum.Pixels
+                };
+            }
```

Comment:
JOh Riienie doee eens normaalk.
Hahahahahahaa.
Dit is Max :-)

Reply:
Dfoeh ff lekler zfel mornaal joh, met je sprakgebrek 🔥 😄